README.md

```diff
@@ -406,7 +406,7 @@ Courses | Duration | Effort | Prerequisites
 These Coursera Specializations all end with a Capstone project.
 Depending on the course, you may be able to utilize the Capstone as your Final Project for this Computer Science curriculum.
 Note that doing a Specialization with the Capstone at the end always costs money.
-So if you don't wish to spend money or use the Capstone as your Final, it may be possible to take the courses in the Specialization for free by manually searching for them, but not all allow this.
+So, if you don't wish to spend money or use the Capstone as your Final, it may be possible to take the courses in the Specialization for free by manually searching for them, but not all allow this.
```

Comment:
Can you alter the commit to remove this change? Unnecessary comma. With that change, happy to merge in the rest.

Reply:
Done, thank you for the review!